### DISCUSSION TITLE
Package versions pinning in requirements

### DISCUSSION THREAD

**Discussion by ExcaliburKG** [open-edge-platform/anomalib#858] [General]
Hi, 
I have a problem with anomalib installation in a complex conda env - onnx==1.10.1 (required when using anomalib extras) - requires an old protobuf version, and causes environment solving error
So the question is: is it neccessary to pin onnx and other packages for anomalib with == and not with <= => ?

**Comment by samet-akcay** [comment]
Hi, of course we need to test it, but you are right. We could loosen the versions.
